sphinx-doc/source/vagrant.rst

```diff
@@ -46,20 +47,22 @@ Note also that VirtualBox `may not be compatible with Docker under windows <http
     ``vagrant suspend``
         Hibernate the container, saving its current state.
 
-    ``vagrant halt``
-        Switch off the container, as if it were a physical computer being shut down.
```

Comment:
Why is `halt` removed? I suppose both `resume` and `halt` would be good to see in this list.

Reply:
For some reason, if I halt the container then when I restart it I can't connect to it over SSH. If I suspend it, it's fine. Did you see anything similar?